{
	"class": "patch",
	"id": "oval:com.ubuntu.xenial:def:46971000000",
	"version": "1",
	"metadata": {
		"title": "USN-4697-1 -- Pillow vulnerabilities",
		"description": "It was discovered that Pillow incorrectly handled certain PCX image files. If a user or automated system were tricked into opening a specially-crafted PCX file, a remote attacker could possibly cause Pillow to crash, resulting in a denial of service. (CVE-2020-35653)  It was discovered that Pillow incorrectly handled certain Tiff image files. If a user or automated system were tricked into opening a specially-crafted Tiff file, a remote attacker could cause Pillow to crash, resulting in a denial of service, or possibly execute arbitrary code. This issue only affected Ubuntu 20.04 LTS and Ubuntu 20.10. (CVE-2020-35654)  It was discovered that Pillow incorrectly handled certain SGI image files. If a user or automated system were tricked into opening a specially-crafted SGI file, a remote attacker could possibly cause Pillow to crash, resulting in a denial of service. This issue only affected Ubuntu 18.04 LTS, Ubuntu 20.04 LTS, and Ubuntu 20.10. (CVE-2020-35655)  Update Instructions:  Run `sudo pro fix USN-4697-1` to fix the vulnerability. The problem can be corrected by updating your system to the following package versions:  python3-pil.imagetk - 3.1.2-0ubuntu1.5 python-pil-doc - 3.1.2-0ubuntu1.5 python3-pil - 3.1.2-0ubuntu1.5 python-pil.imagetk - 3.1.2-0ubuntu1.5 python-imaging - 3.1.2-0ubuntu1.5 python-pil - 3.1.2-0ubuntu1.5 No subscription required",
		"affected": {
			"family": "unix",
			"platform": "Ubuntu 16.04 LTS"
		},
		"reference": [
			{
				"source": "USN",
				"ref_id": "USN-4697-1",
				"ref_url": "https://ubuntu.com/security/notices/USN-4697-1"
			},
			{
				"source": "CVE",
				"ref_id": "CVE-2020-35653",
				"ref_url": "https://ubuntu.com/security/CVE-2020-35653"
			},
			{
				"source": "CVE",
				"ref_id": "CVE-2020-35654",
				"ref_url": "https://ubuntu.com/security/CVE-2020-35654"
			},
			{
				"source": "CVE",
				"ref_id": "CVE-2020-35655",
				"ref_url": "https://ubuntu.com/security/CVE-2020-35655"
			}
		],
		"advisory": {
			"from": "security@ubuntu.com",
			"severity": "Medium",
			"issued": {
				"date": "2021-01-18"
			},
			"cve": [
				{
					"text": "CVE-2020-35653",
					"href": "https://ubuntu.com/security/CVE-2020-35653",
					"priority": "medium",
					"public": "20210112",
					"cvss_score": "7.1",
					"cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:N/A:H",
					"cvss_severity": "high",
					"usns": "4697-1,4697-2"
				},
				{
					"text": "CVE-2020-35654",
					"href": "https://ubuntu.com/security/CVE-2020-35654",
					"priority": "medium",
					"public": "20210112",
					"cvss_score": "8.8",
					"cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
					"cvss_severity": "high",
					"usns": "4697-1"
				},
				{
					"text": "CVE-2020-35655",
					"href": "https://ubuntu.com/security/CVE-2020-35655",
					"priority": "medium",
					"public": "20210112",
					"cvss_score": "5.4",
					"cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:N/A:L",
					"cvss_severity": "medium",
					"usns": "4697-1"
				}
			]
		}
	},
	"criteria": {
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:com.ubuntu.xenial:tst:469710000000",
						"comment": "Long Term Support"
					}
				]
			}
		]
	}
}
